static/js/actions/index.js

```diff
@@ -29,9 +29,9 @@ export const receiveGetUserProfileSuccess = (username, profile) => ({
   payload: { profile, username }
 });
 
-const receiveGetUserProfileFailure = username => ({
+const receiveGetUserProfileFailure = (username, error) => ({
   type: RECEIVE_GET_USER_PROFILE_FAILURE,
-  payload: { username }
+  payload: { username, error }
```

Comment:
Can you rename to `errorInfo` just for consistency with `receiveDashboardFailure`?